### PR TITLE
[API] verify cookie session is iguazio-like sessions

### PR DESCRIPTION
**Pull request by liranbg**

### CODE REVIEW

File: mlrun/api/api/endpoints/frontend_spec.py [hunk 1]
```diff
@@ -44,7 +45,7 @@ def get_frontend_spec(
 ):
     jobs_dashboard_url = None
     session = auth_info.session or session
-    if session:
+    if session and (not auth_info.session and is_iguazio_session_cookie(session)):
```
Comment (quaark) [inline mlrun/api/api/endpoints/frontend_spec.py:48]: why `not auth_info.session`? we might get that session while in iguazio

**Review by quaark — CHANGES_REQUESTED** [review]
Looks good, only one thing

File: mlrun/api/api/endpoints/frontend_spec.py (hunk 1 above)
Comment (liranbg) [reply mlrun/api/api/endpoints/frontend_spec.py]: correct, updated the flow to assume iguazio session/cookie session

**Review by quaark — APPROVED** [review]
LGTM

**Review by Tankilevitch — APPROVED** [review]
Looks good

**Pull request merged**